src/rlx_prv_relup.erl

```diff
@@ -145,7 +155,14 @@ make_upfrom_script(State, Release, UpFrom) ->
                             "relup successfully created!"),
             {ok, State};
         {ok,_, Module,Warnings} ->
-            ?RLX_ERROR({relup_script_generation_warn, Module, Warnings});
+            case WarningsAsErrors of
```

Comment:
i've tried this and apparently no relup is generated in this case so i guess aside from an unclear message there are no major side effects